scripts/run-telestion.sh

```diff
@@ -23,7 +23,10 @@ echo "Pulling Docker Image"
 sudo docker pull "$IMAGE"
 
 echo "Starting the docker image ;-)"
-if sudo docker run -it -p 9870:9870 --privileged --add-host=host.docker.internal:host-gateway --mount "type=bind,source=/data/app-data,target=/usr/telestion/telestion-project-daedalus2/data" --mount "type=bind,source=/usr/bin/nano,target=/usr/bin/nano" --mount "type=bind,source=${SERIAL_UART_DEVICE},target=/dev/telestion-serial" "$IMAGE" "$@"; then 
+if sudo docker run -it -p 9870:9870 --privileged --add-host=host.docker.internal:host-gateway \
+	--mount "type=bind,source=/data/app-data,target=/usr/telestion/telestion-project-daedalus2/data" \
+	--mount "type=bind,source=/usr/bin/nano,target=/usr/bin/nano" \
```

Comment:
questionable whether every host system has nano installed

Reply:
Yes, that are the remnants from the Backstein(tm) :sweat_smile: setup and run script.
We should remove it and put it in the private telestion repository.

Reply:
Done.